### PR TITLE
Update doc for integration test

### PR DESCRIPTION
**Pull request by leakingtapan**

### CODE REVIEW

File: tests/integration/README.md [hunk 1]
```diff
@@ -1,25 +1,18 @@
 ## Integration Testing
+Integration test verifies the functionality of EBS CSI driver as a standalone server outside kuberentes. It exercises the lifecycle of create, attach, stage, mount volume and the reverse operations. And it verifies data can be written onto an EBS volume without any issue.
```
Comment (bertinatto) [inline tests/integration/README.md]: Typo: kuberentes -> Kubernetes

Comment (bertinatto) [reply tests/integration/README.md]: Maybe: `... lifecycle of the volume by creating, attaching...`

File: tests/integration/README.md [hunk 2]
```diff
@@ -1,25 +1,18 @@
...
 
-### Requirements 
+## Run Integration Tests Locally
+Integration test is executed using [aws-k8s-tester](https://github.com/aws/aws-k8s-tester) which is CLI tool for k8s testing on AWS. With aws-k8s-tester, it automates the process of provisioning EC2 instance, pull down and build EBS CSI driver, running the defined integration test and sending test result back. See aws-k8s-tester for more details about how to use it.
```
Comment (bertinatto) [inline tests/integration/README.md]: `The integration test`
`provisioning an EC2 instance, pulling down and building`

File: tests/integration/README.md [hunk 3]
```diff
@@ -1,25 +1,18 @@
...
 
-1. macOS or Linux
-1. `GOPATH` environment variable [set](https://github.com/golang/go/wiki/SettingGOPATH)
-1. [AWS account](https://aws.amazon.com/account/) that has been [configured locally](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+### Requirements
+1. AWS credential is [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+1. Other requirements needed by aws-k8s-tester
```
Comment (bertinatto) [inline tests/integration/README.md]: Maybe a link for `aws-k8s-tester` requirements would be nice (if there's any)

File: tests/integration/README.md [hunk 4]
```diff
@@ -1,25 +1,18 @@
...
 - The master branch of `aws-ebs-csi-driver` is used by default. To run using a pull request for `aws-ebs-csi-driver`, set `PULL_NUMBER` as an environment variable with a value equal to the pull request number.
 
-- When the tests are run, a new VPC is created by default. To run using an existing VPC, set `AWS_K8S_TESTER_VPC_ID` as an environment variable with a value equal to an existing VPC ID.
-
-### Additional Information
-
-- GitHub [repo](https://github.com/aws/aws-k8s-tester) for `aws-k8s-tester`, which includes information about releases and running locally
-- Kubernetes Enhancement Proposal ([KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-aws/20181126-aws-k8s-tester.md)) for `aws-k8s-tester`
\ No newline at end of file
+- When the tests are run, a new VPC is created by default. To run using an existing VPC, set `AWS_K8S_TESTER_VPC_ID` as an environment variable with a value equal to an existing VPC ID. This will be useful when VPC limit is reached in the region under test.
```
Comment (bertinatto) [inline tests/integration/README.md]: It could be useful to have the definition of VPC as well

File: tests/integration/README.md (hunk 3 above)
Comment (leakingtapan) [reply tests/integration/README.md]: There isn't any yet. Created an issue to track this: https://github.com/aws/aws-k8s-tester/issues/32

**Pull request merged**